Packs/Code42/Integrations/Code42/Code42_test.py

```diff
@@ -1292,8 +1391,8 @@ def test_departingemployee_get_all_command_when_no_employees(
     )
     assert cmd_res.outputs_prefix == "Code42.DepartingEmployee"
     assert cmd_res.outputs_key_field == "UserID"
-    assert cmd_res.raw_response == []
-    assert cmd_res.outputs == []
+    assert cmd_res.raw_response == {}
+    assert cmd_res.outputs == {"Results": []}
```

Comment:
Have to do this until PAN fixes their CommandResults to work better with an empty dict as the outputs... This is an issue they are aware of and said that this way works for now.